en/02_Developer_Guides/07_Debugging/01_Error_Handling.md

```diff
@@ -163,12 +163,17 @@ SilverStripe\Core\Injector\Injector:
   LogFileHandler:
     class: Monolog\Handler\StreamHandler
     constructor:
-      - "../silverstripe.log"
+      - "/var/www/silverstripe.log"
       - "info"
 ```
 
-The log file will be relative to the main index.php file path (default: inside public/), so "../silverstripe.log" will
-create a file in your project root.
+[warning]
+The log file path must be an absolute file path, as relative paths may behave differently between CLI and HTTP requests. If you want to use a _relative_ path, you can use the `SS_ERROR_LOG` environment variable to declare a file path that is relative to your project root:
+
+```sh
+SS_ERROR_LOG="./silverstripe.log"
+```
+[/warning]
```

Comment:
So when using the env variable, do we still need the whole yaml config? Or some defaults apply in that case? Or is it a complementary setting just for the log path and needs to be done together with the yaml? I think we need a bit more explanation around how those two things work together or whether they are exclusive to each other etc.

Reply:
Done